### ISSUE TITLE
Unlocker not working on ESXi 6.7U1

### ISSUE THREAD

**Issue by virtualex-itv** [DrDonk/esxi-unlocker#4]
Just upgraded my host to 6.7 Update 1 and ran the unlocker followed by a reboot.  When I attempt to boot my macOS VM, it boot-loops.  Any ideas?

**Comment by amatacchieri** [comment]
Working fine here on an esxi 6.7u01 fresh installation. Thanks mate for that amazing tool!

**Comment by virtualex-itv** [comment]
that's really odd @amatacchieri as I upgraded my hosts and can not get my VM to boot.  My hosts have not had a fresh install since 6.0 and have been upgraded to each release up to the current 6.7U1.  If all you are looking at is if the tool unlocked the features to run macOS, then yes, that part works.  But can you confirm that you're able to boot a macOS VM successfully without a boot loop? I've not been successful...

**Comment by Neek-O** [comment]
Had this problem after updating to the newest build 10764712. Uninstalled the 3.0.0 Unlocker, then reinstalled (and rebooted) and all is well again.

This is a system that started as 6.0 and was upgraded to 6.7 last month via "esxcli software profile update" on the command line.

**Comment by gr4z** [comment]
Fresh install of 6.7u1 and even though the installer reports success, after a reboot the smcPresent=false.

Any suggestions? Thanks

**Comment by mikeyo** [comment]
I too installed a fresh 6.7u1. Created a 10.13 VM, started the VM, stuck on Apple Logo. No errors just wont move past the apple logo. I am using a working 10.13 ISO CD Install. I'm using AMD x399 Threadripper so not sure if that's the cause.

**Comment by DrDonk** [comment]
> I too installed a fresh 6.7u1. Created a 10.13 VM, started the VM, stuck on Apple Logo. No errors just wont move past the apple logo. I am using a working 10.13 ISO CD Install. I'm using AMD x399 Threadripper so not sure if that's the cause.

AMD is not supported by macOS or Unlocker. You will need to find an AMD kernel for macOS.

**Comment by RKGraves** [comment]
Looks like I needed to uncompress and run the ./esxi-install.sh script from a folder off the root of the file system rather than from a sub-folder in a datastore. By creating a /tools folder off of the root, and running the script from there, I am successfully able to run the .sh scripts! 

Thank You for creating this patch!

**Comment by RKGraves** [comment]
(in case someone stumbles on to this thread with a similar problem) A final note to my earlier posts. Turns out the source of the "Operation not Permitted" error with running the ./esxi-install.sh script was due to having UEFI - Secure Boot enabled in the BIOS of the ESXi servers (and nested ESXi servers). Disabling "Secure Boot" in the BIOS allowed the script to run successfully, when copied to a sub-directory on datastore1, on all the remaining training servers. Thanks again for writing and maintaining this patch.

**Comment by virtualex-itv** [comment]
@DrDonk - any updates on this yet to support 6.7U1 Build 10302608 or higher?

**Comment by DrDonk** [comment]
I still can't recreate this error.

**Comment by virtualex-itv** [comment]
@DrDonk - So you’re telling me that you’ve tried installing the unlocker on ESXi 6.7 Update 1 and successfully booted a macOS VM afterwards?  I’m don’t believe that if I have tried numerous times and apparently others have as well and have not been able to boot a VM.  It works on any 6.7 build prior to Update 1 but not on builds of Update 1 or later.  Please help us out

**Comment by DrDonk** [comment]
I have tested the unlocker on 6.7U1 and successfully ran a Snow Leopard VM. I assume you have re-instralled the unlocker after upgrading to U1?

**Comment by virtualex-itv** [comment]
@DrDonk that is correct.  Unlocker was removed prior to upgrading to 6.7 then reinstall post-upgrade and I could not successfully boot my Mojave VM.  I then attempted to install the unlocker on a freshly built ESXi 6.7U1 host, and still could not boot my Mojave VM afterward.  Also, there are others in the thread above that have the same issue.  Seems to work fine on any pre-U1 builds but not on U1 or later builds.

**Comment by DrDonk** [comment]
Can you upload a vmware.log file from your Mojave VM somewhere and post the link here?
